vimiv/gui/metadatawidget.py

```diff
@@ -54,20 +55,42 @@ def __init__(self, parent):
             self._mainwindow_bottom = 0
             self._mainwindow_width = 0
             self._path = ""
-            self._loaded = False
+            self._current_set = ""
 
             api.signals.new_image_opened.connect(self._on_image_opened)
+            api.settings.metadata.current_keyset.changed.connect(self._update_text)
 
             self.hide()
 
         @api.keybindings.register("i", "metadata", mode=api.modes.IMAGE)
         @api.commands.register(mode=api.modes.IMAGE)
-        def metadata(self):
-            """Toggle display of exif metadata of current image."""
+        def metadata(self, count: Optional[int] = None):
+            """Toggle display of exif metadata of current image and handle switching of metadata keysets.
+
+            **count:** Select the key set to display.
+            """
+
```

Comment:
The implementation here works as I would have imagined, perfect!

I feel like we could simplify this a bit though:
1. Simplify the branching by having an overall `if count is not None: ... elif self.isVisible() ... else: ...`
2. Once again relying on the dictionary, replace the `if self._count_is_valid` with a
```
try:
    # The key set switching logic you implemented
except KeyError:
    # Here I would always raise an error message to the user
    raise api.commands.CommandError(f"Invalid key set option {count}")
```